resources/src/main/java/org/robolectric/res/ResType.java

```diff
@@ -54,13 +55,16 @@ public static ResType inferFromValue(String value) {
       try {
         Integer.parseInt(value);
         return INTEGER;
-      } catch (NumberFormatException nfe) {}
+      } catch (NumberFormatException nfe) {
+        Logger.error("Number format exception", nfe);
+      }
 
       try {
         Float.parseFloat(value);
         return FRACTION;
-      } catch (NumberFormatException nfe) {}
-
+      } catch (NumberFormatException nfe) {
+        Logger.error("Number format exception", nfe);
```

Comment:
What about "Failed to infer float from value"?

Reply:
Sure